File: TransCoder_tokenized_test_set_functions/java/FREQUENCY_ELEMENT_UNSORTED_ARRAY.java
static void countFreq ( int [ ] a, int n ) {
  HashMap < Integer, Integer > hm = new HashMap < > ( );
  for ( int i = 0;
  i < n;
  i ++ ) hm . put ( a [ i ], hm . get ( a [ i ] ) == null ? 1 : hm . get ( a [ i ] ) + 1 );
  SortedMap < Integer, Integer > st = new TreeMap < > ( );
  for ( HashMap . Entry < Integer, Integer > x : hm . entrySet ( ) ) {
    st . put ( x . getKey ( ), x . getValue ( ) );
  }
  int cumul = 0;
  for ( SortedMap . Entry < Integer, Integer > x : st . entrySet ( ) ) {
    cumul += x . getValue ( );
    System . out . println ( x . getKey ( ) + " " + cumul );
  }
}

File: TransCoder_tokenized_test_set_functions/python/FREQUENCY_ELEMENT_UNSORTED_ARRAY.py
def countFreq ( a, n ) :
    hm = {
    }
    
    for i in range ( 0, n ) :
        hm [ a [ i ] ] = hm . get ( a [ i ], 0 ) + 1
    st = set ( )
    for x in hm :
        st . add ( ( x, hm [ x ] ) )
    cumul = 0
    for x in sorted ( st ) :
        cumul += x [ 1 ]
        print ( x [ 0 ], cumul )
    
